config/constants.go

```diff
@@ -16,11 +16,33 @@ const (
 // Because Go does not support constant Structs :(
 func defaultConfig() Config {
 	return Config{
-		HistoryFilePath: defaultHistoryFile,
-		MaxHistory:      defaultMaxHist,
-		AllowDuplicates: defaultAllowDuplicates,
-		TempDirPath:     defaultTempDir,
-		LogFilePath:     defaultLogFile,
-		ThemeFilePath:   defaultThemeFile,
+		AllowDuplicates: true,
+		HistoryFilePath: "clipboard_history.json",
+		MaxHistory:      1000,
+		LogFilePath:     "log.txt",
+		ThemeFilePath:   "theme.json",
+		TempDirPath:     "temp",
+		KeyBindings: map[string]string{
+			"filter":        "/",
+			"quit":          "q",
+			"more":          "?",
+			"choose":        "enter",
+			"remove":        "x",
+			"togglePin":     "p",
+			"togglePinned":  "tab",
+			"preview":       " ",
+			"selectDown":    "ctrl+down",
+			"selectUp":      "ctrl+up",
+			"selectSingle":  "s",
+			"clearSelected": "S",
+			"fuzzySelect":   "F",
```

Comment:
fuzzySelect - again not sure what this is. maybe I'm forgetting my own code :laughing: